openocd.cfg

```diff
@@ -1,4 +1,4 @@
-source [find interface/stlink.cfg]
```

Comment:
> echo "WARNING: interface/stlink-v2.cfg is deprecated, please switch to interface/stlink.cfg"

Reply:
Weird. On my machine I don't even have an `stlink.cfg`:
```
> fd stlink /usr/share/openocd/scripts/interface      
/usr/share/openocd/scripts/interface/stlink-v1.cfg
/usr/share/openocd/scripts/interface/stlink-v2-1.cfg
/usr/share/openocd/scripts/interface/stlink-v2.cfg
```
Where is that warning coming from?

Reply:
Newer openocd version.

Reply:
```
> openocd -v
Open On-Chip Debugger 0.10.0
```
I thought 0.10 was the latest version.

Reply:
Yeah, the latest released version. But that is so old that it doesn't cut the mustard anymore. No idea why they're so release unhappy...

Reply:
Alright, how do you want to resolve this? I propose a readme file in the example directory that explains the issue and a comment in `openocd.cfg`. I'd prefer keeping `stlink-v2.cfg` in there, though. I'm worried that people encounter the same problem as I did when they just install their distro's openocd package. In my opninion, the example should work without tinkering and those that have manually installed a more recent OpenOCD version probably know how to handle the warning you posted.
Let me know what you prefer and I'll make the changes.

Reply:
It's a lose-lose situation and I don't really care. Let's go with this then.